src/api/spec/controllers/staging/staging_projects_controller_spec.rb

```diff
@@ -97,7 +97,7 @@
             assert_select 'entry', 2
           end
           assert_select 'missing_reviews', 1 do
-            assert_select 'entry', 1
+            assert_select 'entry', 4
```

Comment:
hmm, so it's reporting the staging manager group's reviews as missing reviews? That sounds like a bug.

Reply:
can you track this please? It's unrelated to this PR, but it shouldn't be dropped.

Reply:
looks like there is no need as your test fails now for being 1 :)

Reply:
@coolo, after rebasing  [this PR](https://github.com/openSUSE/open-build-service/pull/6721), everything changed. Caused by `skip_sanitize` on [this line](https://github.com/coolo/open-build-service/blob/908bf1d850bbd9eb2dcfcccd8e3ef40c023f8601/src/api/spec/factories/bs_requests.rb#L17)

I'm going to fix all related with this PR first and then _missing_reviews._

Reply:
the reviews are created in the factory, see https://github.com/coolo/open-build-service/blob/908bf1d850bbd9eb2dcfcccd8e3ef40c023f8601/src/api/spec/factories/bs_requests.rb#L24